content/consul/v1.21.x/content/docs/secure/sso/auth0.mdx

```diff
@@ -7,86 +7,28 @@ description: >-
 
 # Enable Auth0 single sign-on (SSO)
 
-This page describes the process to configure Consul to use Auth0 for single sign-on (SSO) in the Consul UI and CLI.
+This page describes the process to configure Consul to use Auth0 for single sign-on (SSO) in the Consul UI and CLI using [OIDC](https://en.wikipedia.org/wiki/OpenID_Connect).
+
+OIDC authentication is useful when you want to deploy SSO widely in your organization and do not want to manage access with individual Consul ACL tokens. Once implemented, SSO will enable an interactive login procedure that can be initiated from either the Consul UI or the command line.
+
+In this example you will configure [Auth0](https://auth0.com/) as an identity provider, and use the user metadata in Auth0 to automatically grant permissions in Consul ACL. This approach unifies the user permissions grant without the need to create individual user tokens or to define permissions individually for the different users.
```

Comment:
```suggestion
These instructions configure Auth0 as an identity provider, so that Consul can use data in Auth0 to automatically grant permissions for Consul ACL. This approach unifies the user permissions grants without creating individual user tokens or manually defining individual permissions for different users.
```